client/src/App.tsx

```diff
@@ -18,6 +18,7 @@ export default function App() {
   const [user, setUser] = useState<User>();
```

Comment:
Line 17:
With useState, you don't have to add `| undefined` if you call useState with no arguments since it will get added for you automatically by TS. It doesn't hurt to have it, though.

For example, on line 18, if you hover over `user` you will see that it is typed as `User | undefined` because `useState` is called without arguments.